README.md

```diff
@@ -187,6 +187,25 @@ Note that only the [JSON0 OT type](https://github.com/ottypes/json0) is supporte
 `share.close(callback)`
 Closes connections to the database and pub/sub adapters.
 
+### Logging
+
+ShareDB logs some errors and warnings, which can be useful for debugging. By default, the log level is set to `warn`. This can be configured through the `logLevel` constructor argument, which can be set to any of the following:
```

Comment:
Looks like the default is actually set to `logger.levels.ERROR`?

Reply:
Great point!

Reply:
@ericyhwang setting it to `WARN` to keep in-line with the `loglevel` default, and to not swallow too much.